tests/test_browser.py

```diff
@@ -3579,6 +3579,14 @@ def test_pthread_gauge_available_memory(self):
       for args in [[], ['-s', 'USE_PTHREADS=1']]:
         self.btest(path_from_root('tests', 'gauge_available_memory.cpp'), expected='1', args=['-s', 'ABORTING_MALLOC=0'] + args + opts)
 
+  # Test PTHREAD_WORKER_PRE_JS
```

Comment:
This comment seems a little redundant with the test name. Maybe unpack it a litle, "Test that users can prepend JS to workers" or something

Reply:
Sounds good, will update.